Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/signature/core/DccStateChecker.kt

```diff
@@ -22,10 +25,29 @@ class DccStateChecker @Inject constructor(
     suspend fun checkState(
         dccData: DccData<*>
     ): Flow<CwaCovidCertificate.State> = flow {
-        // TODO
-        val state = CwaCovidCertificate.State.Valid(
-            expiresAt = Instant.now().plus(Duration.standardDays(21))
-        )
+        // TODO signature check
```

Comment:
Will this be done in a sepparate PR?

Reply:
Part of https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8361